docs/userguide/modules/getting-started/pages/installing-service-binding.adoc

```diff
@@ -116,33 +115,33 @@ service-binding-operator-helm-chart          	https://redhat-developer.github.io
 ----
 The output verifies that the `service-binding-operator-helm-chart` repository is added to your local helm repository.
 
-[## Installing the Helm chart]
+[##installing-the-helm-chart]
```

Comment:
From L118 to L173:
[#installing-the-helm-chart]
== Installing the Helm chart

. Search the repository:
+
[source,bash]
----
helm search repo service-binding-operator-helm-chart
----
+
.Example output
[source,terminal]
----
NAME                                                            CHART VERSION	APP VERSION	DESCRIPTION                                    
service-binding-operator-helm-chart/service-binding-operator    1.0.0        	1.0.1      	A Helm chart to deploy service binding operator
----
+
. Create a Helm chart release and specify the namespace required with the `--create-namespace` flag:
+
[source,bash]
----
helm install service-binding-operator-release \
service-binding-operator-helm-chart/service-binding-operator \
--namespace service-binding-operator --create-namespace
----
+
. Optional: If you wish to install the chart on the default namespace, remove the `--namespace` and `--create-namespace` flags.
+
[NOTE]
====
As part of the Helm test, objects such as deployment, service binding resources, and secrets used for testing the Operator are deleted.
====

. Optional: To view the  resources created for testing, install the chart with the `keepTestResources` flag value set to `true`:
+
[source,bash]
----
helm install service-binding-operator-release \
service-binding-operator-helm-chart/service-binding-operator \
--namespace service-binding-operator --create-namespace \
--set keepTestResources=true
----
+
.Example output
[source,terminal]
----
NAME: service-binding-operator-release
LAST DEPLOYED: Mon May 16 09:15:16 2022
NAMESPACE: service-binding-operator
STATUS: deployed
REVISION: 1
----

. Verify that the chart is succesfully installed:
+
[source,bash]
----
kubectl get pods --namespace service-binding-operator
----
+
.Example output
[source,terminal]
----
NAME                                        READY   STATUS    RESTARTS   AGE
service-binding-operator-78c6444b4d-kswhk   1/1     Running   0          21s
----

Reply:
Just copy-paste whatever I wrote above, for  L118 to L173: